src/components/picker/index.js

```diff
@@ -321,10 +323,12 @@ class Picker extends BaseComponent {
     }
 
     const textInputProps = TextField.extractOwnProps(this.props);
+    const style = textInputProps.style;
```

Comment:
style is a common prop, all components has it, it has no meaning getting it from `textInputProps`.

Reply:
Removed